server/src/test/java/com/epam/aidial/core/server/controller/DeploymentPostControllerTest.java

```diff
@@ -169,8 +170,10 @@ public void testDeploymentIsNotAccessible() {
         verify(context).respond(eq(FORBIDDEN), anyString());
     }
 
+    @Disabled
```

Comment:
?

Reply:
I consulted with @astsiapanay about that. Two tests I marked disabled did not reflect the actual code state. We need to rewrite them if needed.

Reply:
Do they pass? If not, why do they suddenly start failing?

Reply:
They did not pass because dependencies were insufficient mocks.

Reply:
`   when(context.getDeployment()).thenReturn(application);`
That is a mock used by the tests. However, it is not the way how real info about deployment obtained in post controller.